### PR TITLE
Update to Android SDK v4.16

### PR DESCRIPTION
**Pull request by Redth**

### CODE REVIEW

File: Facebook.Android/nuget/Xamarin.Facebook.Android.nuspec [hunk 1]
```diff
@@ -16,6 +16,7 @@
       <group targetFramework="MonoAndroid403">
```
Comment (bholmes) [inline Facebook.Android/nuget/Xamarin.Facebook.Android.nuspec]: I am having a hard time matching up the NuGet dependency list with Xamarin.Facebook/packages.config.  Should they not be the same?

File: Facebook.Android/nuget/Xamarin.Facebook.AudienceNetwork.Android.nuspec [hunk 2]
```diff
@@ -15,6 +15,7 @@
     <dependencies>
```
Comment (bholmes) [inline Facebook.Android/nuget/Xamarin.Facebook.AudienceNetwork.Android.nuspec]: Same questions here.  This list does not match Xamarin.Facebook.AudienceNetwork/packages.config.

File: Facebook.Android/component/component.yaml [hunk 3]
```diff
@@ -15,8 +15,8 @@ libraries:
     - ../output/Xamarin.Facebook.AudienceNetwork.dll
 packages:
   android:
```
Comment (bholmes) [inline Facebook.Android/component/component.yaml]: Probably easier to just list the 2 Xamarin.Facebook.* NuGets here and let NuGet dependency do its job.

File: Facebook.Android/nuget/Xamarin.Facebook.Android.nuspec (hunk 1 above)
Comment (Redth) [inline Facebook.Android/nuget/Xamarin.Facebook.Android.nuspec]: I suppose we could add the implied dependencies explicitly...   In this case, AppCompat-v7 pulls in the Vector and Animated Vector drawable packages... the nuspec doesn't explicitly list these.

File: Facebook.Android/nuget/Xamarin.Facebook.AudienceNetwork.Android.nuspec (hunk 2 above)
Comment (Redth) [inline Facebook.Android/nuget/Xamarin.Facebook.AudienceNetwork.Android.nuspec]: Similar situation here as above, the extra dependencies you see in packages.config get pulled in as sub-dependencies of packages which is why they don't match.

File: Facebook.Android/component/component.yaml (hunk 3 above)
Comment (Redth) [inline Facebook.Android/component/component.yaml]: Yeah this is being unnecessarily explicit.  The other dependencies will come in automatically.  Will simplify this bit.

**Pull request merged**